### ISSUE TITLE
Preview not working correctly after v0.2.0 in Design View

### ISSUE THREAD

**Issue by chathurabuddi** [facebook/shimmer-android#100]
This is not rendering correctly after upgrading from `v0.1.0` to `v0.5.0`. It is showing a gray color box instead of actual elements inside the layout.

**Design View Preview with ShimmerFrameLayout v0.1.0**
![1](https://user-images.githubusercontent.com/10394110/83388583-4f95c680-a40c-11ea-8b20-96b6e3014877.png)

**Design View Preview with ShimmerFrameLayout v0.5.0**
![2](https://user-images.githubusercontent.com/10394110/83388604-5a505b80-a40c-11ea-9cf2-490a780374e2.png)

**Comment by mengxn** [comment]
base alpha is 0.3 default in version 0.5.0, you can change it in xml
`app:shimmer_base_alpha="0.3"`

**Comment by chathurabuddi** [comment]
> base alpha is 0.3 default in version 0.5.0, you can change it in xml
> `app:shimmer_base_alpha="0.3"`

No, if I use `app:shimmer_base_alpha="1"`, there is no shimmer animation shown when running.

**Comment by mengxn** [comment]
you should config this
```
// apply custom color
app:shimmer_colored="true"
app:shimmer_base_color="color"
app:shimmer_highlight_color="color"
```

**Comment by chathurabuddi** [comment]
@mengxn, This will change the base and highlight colors, But I don't want to change default colors. In the v0.1.0, the preview worked without any issue.

**Comment by CodeIdeal** [comment]
why not use  tools namespace?

**Comment by chathurabuddi** [comment]
> why not use tools:xxx namespace?

can you please elaborate more on this?

**Comment by xiphirx** [comment]
What was the behavior like for 0.1.0? We didnt add specific editor support so I'm not sure what you're expecting it to look like.

**Comment by konstantinos-apostolakis** [comment]
Did you find any solution to this?
